src/components/Categories.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 
-const Categories = () => (
-  <button type="button">Check Status</button>
-);
+const Categories = () => {
+  const categories = useSelector((state) => state.categories);
+  return (
+    <div>
+      <h1>{categories}</h1>
+      <button type="button">Check Status</button>
```

Comment:
- [ ] Please add an action to your redux store such that when I click the check status button, it returns a message for example "Under Construction". It is a requirement stated in the milestone description 👍